main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -143,7 +144,8 @@ object CompletionProvider {
         case None => ""
         case Some(s) => getLastWord(s)
       }
-      val range = Range(Position(y, start), Position(y, end))
+      // Remember positions are one-indexed.
+      val range = Range(Position(y+1, start+1), Position(y+1, end+1))
```

Comment:
This is the only use of line/character that was iffy. I think this was wrong before, since stores positions back with zero-indexing. Its only used by `getHoleCompletion` so might've been unnoticed.

Reply:
Can you test it?

Reply:
I can't get holeCompletion to trigger on master or this PR :L

Reply:
Try to force autocomplete on top of the hole.

Reply:
I did, `no suggestions`